packages/artifact/__tests__/upload.test.ts

```diff
@@ -102,7 +102,7 @@ describe('Upload Tests', () => {
       uploadHttpClient.createArtifactInFileContainer(artifactName)
     ).rejects.toEqual(
       new Error(
-        `Unable to create a container for the artifact invalid-artifact-name at ${getArtifactUrl()}`
+        `Create Artifact Container failed: Artifact service responded with 400 : The artifact name invalid-artifact-name is not valid. Request URL ${getArtifactUrl()}`
```

Comment:
Personally, I am not a fan of showing users the error code - that feels like implementation detail and doesn't help user to fix the issue.  Also we could change the server side error code later, and if we show it here, creates an extra place we need to update.

Reply:
In this case I feel like it's harmless to say it's a 400. Most action authors are tech-say so this will help them know that this was caused by them

Reply:
Switched it up to not mention the 400